lib/SILGen/SILGenExpr.cpp

```diff
@@ -2384,17 +2384,17 @@ visitInterpolatedStringLiteralExpr(InterpolatedStringLiteralExpr *E,
   RValue interpolation;
   {
     TapExpr *ETap = E->getAppendingExpr();
+
     // Inlined from TapExpr:
```

Comment:
I will try to clean up this duplication tomorrow.

Reply:
I was able to improve the factoring a little bit, but I had trouble actually removing the duplication because it reorders code in a way that disrupts the OSLogOptimization pass. I'll tag Ravi in a follow-up PR and see if we can figure something out.